Bottom: c9a943dcf902b0bbe9c8e99e542b8779d61c38f2
Top:    ecc9ecc78540c6d88ebe3cf1a28cb6218d094387
Author: Ian Kent <raven@themaw.net>
Date:   2013-05-02 16:09:42 +0800

Refresh of autofs-5.0.7-use-MS_UNBINDABLE-for-autofs-mounts.patch

---

diff --git a/daemon/direct.c b/daemon/direct.c
index 5c1006d..ffa4580 100644
--- a/daemon/direct.c
+++ b/daemon/direct.c
@@ -37,6 +37,8 @@
 #define INCLUDE_PENDING_FUNCTIONS
 #include "automount.h"
 
+extern unsigned long mountflags;
+
 /* Attribute to create detached thread */
 extern pthread_attr_t th_attr_detached;
 
diff --git a/daemon/indirect.c b/daemon/indirect.c
index b2b4bce..b0ce508 100644
--- a/daemon/indirect.c
+++ b/daemon/indirect.c
@@ -36,6 +36,8 @@
 #define INCLUDE_PENDING_FUNCTIONS
 #include "automount.h"
 
+extern unsigned long mountflags;
+
 /* Attribute to create detached thread */
 extern pthread_attr_t th_attr_detached;
 
diff --git a/include/automount.h b/include/automount.h
index 46e8530..40481f0 100644
--- a/include/automount.h
+++ b/include/automount.h
@@ -80,8 +80,6 @@ int load_autofs4_module(void);
 #define MS_UNBINDABLE (1 << 17)
 #endif
 
-extern unsigned long mountflags;
-
 /* This sould be enough for at least 20 host aliases */
 #define HOST_ENT_BUF_SIZE	2048
